src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -1975,18 +1975,24 @@ put_unwind_info(unw_addr_space_t as, unw_proc_info_t *pip, void *arg)
 {
 }
 
-static unw_accessors_t unwind_accessors =
+static unw_accessors_t init_unwind_accessors()
```

Comment:
Can you please change it as follows? That way, no function call will happen and the compiler will initialize the fields at compile time.
```suggestion
static constexpr unw_accessors_t init_unwind_accessors()
```

Reply:
Reverted.  See below.